src/deps_log.cc

```diff
@@ -218,9 +217,9 @@ bool DepsLog::Load(const string& path, State* state, string* err) {
       assert(size % 4 == 0);
       int* deps_data = reinterpret_cast<int*>(buf);
       int out_id = deps_data[0];
-      int mtime = deps_data[1];
-      deps_data += 2;
-      int deps_count = (size / 4) - 2;
+      int64_t mtime = reinterpret_cast<int64_t*>(&deps_data[1])[0];
```

Comment:
Probably, I was trying to avoid thinking about endianness, but I'll have to see if I can implement this in some way that's not annoying.